lib/matplotlib/testing/_markers.py

```diff
@@ -8,6 +8,7 @@
 import pytest
 
 import matplotlib.testing
+import matplotlib.testing.compare
```

Comment:
Is this required?

Reply:
Yes, we use `matpotlib.testing.compare` in this module.  Under "normal" usage of running all of the tests, something else imports `matplotlib.testing.compare` before the code in this module gets run so it works.  However if you run just one module of tests (iirc, it was `test_backend_ps`) then that import does not happen an you get an `AttributeError`.